### PR TITLE
Wemadeit

### PR DESCRIPTION
**Pull request by ColeHassett**
pls i promise it's good

### CODE REVIEW

File: bot.go [hunk 1]
```diff
@@ -1,10 +1,61 @@
...
+		return
+	}
+
+	dg.AddHandler(func(s *discordgo.Session, event *discordgo.Ready) {
+		s.ChannelMessageSend("522472817283956745", "FARTS WE GO IN")
+	})
+	dg.AddHandler(messageCreate)
+
+	// Receive message events
+	// THIS IS DOGSHIT
```
Comment (colinthatcher) [inline bot.go:39]: this comment is not very good. Could you please revise it

File: bot.go [hunk 2]
```diff
@@ -1,10 +1,61 @@
...
+	dg.Identify.Intents = discordgo.IntentsGuilds | discordgo.IntentsGuildMessages
+
+	// Open websocket connection to discord
+	dg.Open()
```
Comment (colinthatcher) [inline bot.go:43]: I think you are leaving an error unchecked here. Please check it

File: bot.go [hunk 3]
```diff
@@ -1,10 +1,61 @@
...
+	}
+
+	dg, err := discordgo.New("Bot " + *BotToken)
+	if err != nil {
+		fmt.Println("YOU RUINED IT: ", err)
+		return
+	}
+
+	dg.AddHandler(func(s *discordgo.Session, event *discordgo.Ready) {
+		s.ChannelMessageSend("522472817283956745", "FARTS WE GO IN")
```
Comment (colinthatcher) [inline bot.go:34]: This string should probably not be hard coded

**Review by colinthatcher — APPROVED** [review]

File: bot.go (hunk 1 above)
Comment (ColeHassett) [reply bot.go:39]: no

**Pull request merged**